src/Agent.Worker/DiagnosticLogManager.cs

```diff
@@ -105,6 +105,37 @@ public sealed class DiagnosticLogManager : AgentService, IDiagnosticLogManager
                 File.Copy(agentLogFile, destination);
             }
 
+            // Read and add to logs waagent.conf settings on Linux
+            if (PlatformUtil.RunningOnLinux)
+            {
+                executionContext.Debug("Dumping of waagent.conf file");
+                string waagentDumpFile = Path.Combine(supportFilesFolder, "waagentConf.txt");
+
+                string configFileName = "waagent.conf";
```

Comment:
Do you know if this file could be located by some custom path?

Reply:
Right now waagent support only this standard path. Right now I don't see any ticket in https://github.com/Azure/WALinuxAgent with requests to support custom paths